### PR TITLE
♻️ REFACTOR: EBP Migration

### PR DESCRIPTION
**Pull request by chrisjsewell**
Initial changes:

- Move tests outside package
- Change links in README and add contributing section
- Add extras to setup.py
- Add `__version__` to package and read in setup.py
- Add tox.ini configuration
- Add and run pre-commit configuration
- Pin python to >=3.6,<4
- Pin sphinx to >=2,<4
- Add GitHub CI workflow (removed TravisCI hook)
- remove built docs folder and instead build docs with RTD (and test in PRs)

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: continuous-integration
+
+on:
+  push:
+    branches: [master]
+    tags:
+      - 'v*'
+  pull_request:
```
Comment (Daltz333) [inline .github/workflows/tests.yml:11]:
```suggestion
  pull_request:
    branches: [master]
```

File: README.md [hunk 2]
```diff
@@ -24,10 +24,10 @@ extensions = ['sphinx_tabs.tabs']
 
 If you are using [Read The Docs](https://readthedocs.org/) for building your documentation, the extension must be added as a requirement. Please add the following to `requirements.txt` at the root of the project:
 ```
-https://github.com/djungelorm/sphinx-tabs/releases/download/v1.1.12/sphinx-tabs-1.1.13.tar.gz
-``` 
+https://github.com/executablebooks/sphinx-tabs/releases/download/v1.1.12/sphinx-tabs-1.1.13.tar.gz
```
Comment (Daltz333) [inline README.md:27]: You can now just use the Pypi release

```suggestion
sphinx-tabs==1.1.13
```

File: .github/workflows/tests.yml (hunk 1 above)
Comment (chrisjsewell) [reply .github/workflows/tests.yml:11]: any specific reason to restrict to master?

Comment (Daltz333) [reply .github/workflows/tests.yml:11]: No particular reason. I just thought it would only run if you specified a target branch.

Comment (chrisjsewell) [reply .github/workflows/tests.yml:11]: no it runs on all branches if you don't specify 😄

**Comment by chrisjsewell** [comment]
Ok I'm done with this, @djungelorm would you like to have a look before I merge?

**Comment by chrisjsewell** [comment]
Going to go ahead and merge this in an hour or two 😬

**Comment by Daltz333** [comment]
It looks pretty good to me.

**Comment by chrisjsewell** [comment]
ok lets go for it; FYI after this I'm going to add https://black.readthedocs.io to the pre-commit (and run for all the files), and at some point I'd also like to convert the tests to use pytest fixtures

**Pull request merged**